robottelo/test.py

```diff
@@ -416,6 +413,22 @@ def _restore_from_savepoint(self, savepoint):
         self.logger.info('Reset db from /home/backup/{}'.format(savepoint))
         ssh.command('./reset-db.sh /home/backup/{}'.format(savepoint))
 
+    def _get_subscription_id(self):
+        """Get subscription id"""
+        result = Subscription.list(
+            {'organization-id': self.org_id},
+            per_page=False
+        )
+
+        if result.return_code != 0:
+            self.logger.error('Fail to get subscription id!')
+            return
+        subscription_id = result.stdout[0]['id']
+        subscription_name = result.stdout[0]['name']
+        self.logger.info('Subscribed to {0} with subscription id {1}'
+                         .format(subscription_name, subscription_id))
```

Comment:
For the sake of conformity, please indent code instead of using hanging blocks.

```python
        self.logger.info(
            'Subscribed to {0} with subscription id {1}'
            .format(subscription_name, subscription_id)
        )
```